### PR TITLE
Add coherent animations on all elements using SASS placeholder

### PR DESCRIPTION
**Pull request by lkulbacki**
Covers: https://projects.kodilla.com/browse/WDP181201-9

Should cover all animations in current state of the page (master branch). Does not cover animating submenu, implemented in this PR: https://github.com/OskarLebuda/wdp-1812-01/pull/4, remaining task noted in JIRA: https://projects.kodilla.com/browse/WDP181201-38.

### CODE REVIEW

File: src/sass/_mixin_extend.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
+%animate {
+  // in place of
+  // transition: all ease-in-out 250ms;
+  // added separate transitions to control the transition time and animation separately for each property
+  transition: background-color ease-in-out 250ms,
+  color ease-in-out 250ms,
```
Comment (pawel-schmidt) [inline src/sass/_mixin_extend.scss]: As a continuation of `transition` value, you should have these lines indented.

File: src/sass/_mixin_extend.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  visibility ease-in-out 250ms,
+  opacity ease-in-out 250ms,
+  border ease-in-out 250ms;
+
+}
+
+a {
```
Comment (pawel-schmidt) [inline src/sass/_mixin_extend.scss]: Move this `a` definition below `body` in `style.scss`.

File: src/sass/_mixin_extend.scss (hunk 1 above)
Comment (lkulbacki) [reply src/sass/_mixin_extend.scss]: WebStorm autoformatter keeps them like that, can't change that option. But I changed that by hand.

File: src/sass/_mixin_extend.scss (hunk 2 above)
Comment (lkulbacki) [reply src/sass/_mixin_extend.scss]: Done.

**Comment by lkulbacki** [comment]
@pawel-schmidt changes applied as requested

**Pull request merged**